### PR TITLE
Update splink demos

### LINKED ISSUE #1332: Error messages printed in docs
### What happens?

Error messages displaying in code output blocks

### To Reproduce

See Examples > Pyspark > Deduplication using Pyspark section of docs here: https://moj-analytical-services.github.io/splink/demos/example_simple_pyspark.html

### OS:

iOS

### Splink version:

3.9.1

### Have you tried this on the latest `master` branch?

- [X] I agree

### Have you tried the steps to reproduce? Do they include all relevant data and configuration? Does the issue you report still appear there?

- [X] I agree

### PR DESCRIPTION
**Pull request by RossKen**
### Type of PR

- [ ] BUG
- [ ] FEAT
- [ ] MAINT
- [X] DOC

### Is your Pull Request linked to an existing Issue or Pull Request?

Closes #1332 

Related splink demos PR https://github.com/moj-analytical-services/splink_demos/pull/92

### Give a brief description for the solution you have provided

The tutorial and example notebooks are in need of an update, and bringing mkdocs.yml into line with new folder structure of splink_demos. In addition, the rendering of charts has been broken for a while so that also needs fixed.

### PR Checklist

- [ ] Added documentation for changes
- [ ] Added feature to example notebooks at tutorial in [splink_demos](https://github.com/moj-analytical-services/splink_demos) (if appropriate)
- [ ] Added tests (if appropriate)
- [ ] Made changes based off the latest version of Splink
- [ ] Run the [linter](https://moj-analytical-services.github.io/splink/dev_guides/changing_splink/lint.html)

### CODE REVIEW

**Comment by RossKen** [comment]
The scope of this PR has been scaled back to just manage the changing file structure of splink_demos. The rendering will be dealt with separately after the migration of tutorials and examples to this repo

**Comment by RossKen** [comment]
@ThomasHepworth given the splink_demos PR has been merged, can you give this a quick approve to update the folder structure here

**Review by ThomasHepworth — APPROVED** [review]
I've double checked and the links all look correct.

File: mkdocs.yml [hunk 1]
```diff
@@ -83,30 +83,32 @@ nav:
...
-        - Linking two tables of persons: "demos/example_link_only.ipynb"
-        - Real time record linkage: "demos/example_real_time_record_linkage.ipynb"
-        - QA from ground truth column: "demos/example_accuracy_analysis_from_labels_column.ipynb"
-        - Estimating m probabilities from labels: "demos/example_pairwise_labels.ipynb"
-        - Quick and dirty persons model: "demos/example_quick_and_dirty_persons.ipynb"
-        - Febrl3 Dedupe: "demos/example_febrl3.ipynb"
-        - Febrl4 link-only: "demos/example_febrl4.ipynb"
+        - Deduplicate 50k rows historical persons: "demos/examples/duckdb/deduplicate_50k_synthetic.ipynb"
+        - Linking financial transactions: "demos/examples/duckdb/transactions.ipynb"
+        - Linking two tables of persons: "demos/examples/duckdb/link_only.ipynb"
```
[the remark this answers is not in the document]
Comment (RossKen) [reply mkdocs.yml:99]: Ah, yea that was an oversight on my part

**Comment by RossKen** [comment]
> I've double checked and the links all look correct.

Thanks!

File: mkdocs.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (RossKen) [reply mkdocs.yml:99]: will add before merging

**Pull request merged**